src/benchmarkstt/modules.py

```diff
@@ -35,3 +35,65 @@ def __getitem__(self, key):
 
     def keys(self):
         return [key for key, value in iter(self)]
+
+
+def load_object(name, transform=None):
+    """
+    Load an object based on a string.
+    :param name: The string representation of an object
+    :param transform: Transform (callable) done on the object name for comparison, if None, will lowercase compare.
+    False for no transform.
+    """
+    module = list(name.split('.'))
+
+    if transform is None:
+        transform = str.lower
+    elif transform is False:
+        def identity(x):
+            return x
+        transform = identity
+
+    class_name = transform(module.pop())
+    module = '.'.join(module)
+    if len(module) == 0:
+        module = globals()
+    else:
+        module = import_module(module)
+
+    for found_class_name in dir(module):
+        if transform(found_class_name) != class_name:
+            continue
+        return getattr(module, found_class_name)
+
+    raise ImportError("Could not find an object for %r" % (name,))
+
+
+class Proxy:
+    """
+    Pass all function calls to instance, or to parent class if instance does not
+    implement it.
+    """
```

Comment:
This looks like some dark magic. I like the idea! Looking forward to seeing how it's going to be used. I don't think there are any usages of this class yet, right?

Reply:
Dark magic indeed... you can find an example of it’s usefulness at the bottom of normalization/core.py commented out.